client/src/components/Layout/Layout.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useLocation } from "react-router-dom";
+
+import "./Layout.css";
+import Navbar from "../../components/Navbar/Navbar"
+import Footer from "../../components/Footer/Footer"
+
+
+export default function Layout(props) {
+    let currentPath = useLocation().pathname.split('/')[1]
+    console.log(currentPath)
+
+    const pathPages = {
+        '': 99,
```

Comment:
🤔🤔🤔🤔🤔🤔

Reply:
The way the Navbar loads, if a page is out of range, then the triangle highlighter is not applied.  The previous code had the page set to 5, which is why you didn't notice it.  I set it to 99, so it is obvious which pages should not receive triangle.  This also allows for future nav menu items to be added without requiring code refactoring.

![image](https://user-images.githubusercontent.com/58237351/87327675-5aab3e80-c502-11ea-978e-2bdb0152fcf3.png)